README.md

```diff
@@ -64,3 +56,51 @@ How to contribute ?
 - [Preethi G](https://github.com/preethical)
 - [Shivam Sharma](https://github.com/shivamragnar)
 - [Shreya Agrawal](https://github.com/shreyaagrawal0809)
+
+Folder structure
+----------------
+
+```
+.
+├── LICENSE
+├── README.md
+├── build
+│   ├── favicon.ico
+│   ├── logo192.png
```

Comment:
don't need to name last level files.

Reply:
Made a few changes in the PR, you can have a look @TheDataAreClean.